src/AzureBlobStorageAdapter.php

```diff
@@ -277,9 +277,14 @@ public function copy(string $source, string $destination, Config $config): void
 
     public function temporaryUrl(string $path, \DateTimeInterface $expiresAt, Config $config): string
     {
+        $permissions = $config->get("permissions", "r");
```

Comment:
Wouldnt be an enum a better idea instead of a plain string here?

Not that its wrong but it would not bite us in the future

Reply:
There already is a helper to generate these strings

(string) new BlobContainerSasPermissions(read: true, write: true)